tests/JsonLd/Action/ContextActionTest.php

```diff
@@ -82,4 +83,32 @@ public function testContextActionWithThrown()
         );
         $contextAction('dummy');
     }
+
+    public function testContextActionWithFormatThrown()
+    {
+        $this->expectException(BadRequestHttpException::class);
+
+        $contextBuilderProphecy = $this->prophesize(ContextBuilderInterface::class);
+        $resourceNameCollectionFactoryProphecy = $this->prophesize(ResourceNameCollectionFactoryInterface::class);
+        $resourceMetadataFactoryProphecy = $this->prophesize(ResourceMetadataFactoryInterface::class);
+        $resourceNameCollectionFactoryProphecy->create()->willReturn(new ResourceNameCollection(['gerard']));
+        $contextAction = new ContextAction($contextBuilderProphecy->reveal(), $resourceNameCollectionFactoryProphecy->reveal(), $resourceMetadataFactoryProphecy->reveal());
+
+        $contextAction('dummy.html');
+    }
+
+    public function testContextActionWithResourceClassAndFormat()
+    {
+        $contextBuilderProphecy = $this->prophesize(ContextBuilderInterface::class);
+        $resourceNameCollectionFactoryProphecy = $this->prophesize(ResourceNameCollectionFactoryInterface::class);
+        $resourceMetadataFactoryProphecy = $this->prophesize(ResourceMetadataFactoryInterface::class);
+        $resourceNameCollectionFactoryProphecy->create()->willReturn(new ResourceNameCollection(['dummy']));
+        $contextAction = new ContextAction($contextBuilderProphecy->reveal(), $resourceNameCollectionFactoryProphecy->reveal(), $resourceMetadataFactoryProphecy->reveal());
+        $contextBuilderProphecy->getResourceContext('dummy')->willReturn(['/dummies']);
+
+        $resourceMetadataFactoryProphecy->create('dummy')->shouldBeCalled()->willReturn(
+            new ResourceMetadata('dummy', 'dummy', '#dummy', ['get' => ['method' => 'GET'], 'put' => ['method' => 'PUT']], ['get' => ['method' => 'GET'], 'post' => ['method' => 'POST'], 'custom' => ['method' => 'GET', 'path' => '/foo'], 'custom2' => ['method' => 'POST', 'path' => '/foo']], [])
+        );
+        $this->assertEquals(['@context' => ['/dummies']], $contextAction('dummy.jsonld'));
```

Comment:
Could you please also add a test with `$contextAction('dummy')` for non-regression?

Reply:
there's already one.